android/src/main/java/com/amplitude/android/Amplitude.kt

```diff
@@ -72,8 +84,21 @@ open class Amplitude(
         add(AnalyticsConnectorIdentityPlugin())
         add(AnalyticsConnectorPlugin())
         add(AmplitudeDestination())
+        val plugins = configuration.plugins
```

Comment:
I like the idea of adding plugins through the configurations!
The order of the plugins also matters? should we support ordering the plugins?

Reply:
Order is the same as current. Plugins run ordered by type, and then FIFO per type